Dockerfile

```diff
@@ -34,7 +35,8 @@ COPY conf/jenkins/nodeMonitors.xml "${JENKINS_STAGING}/nodeMonitors.xml"
 # Override the default property for DNS lookup caching
 RUN echo 'networkaddress.cache.ttl=60' >> ${JAVA_HOME}/jre/lib/security/java.security
 
-CMD /usr/local/jenkins/bin/bootstrap.py && nginx && \
+CMD . /usr/local/jenkins/bin/export-libssl.sh && \
```

Comment:
Yes. It did not like `source`, but accepted `.`. This is to source the file `export-libssl.sh` and have its variables used for the rest of the commands.